### PR TITLE
Add cypress to prefix path router

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

**Comment by sneridagh** [comment]
Ok, it does work :) Tests are failing because URL checks that do not expect prefixed paths.

Rather than duplicate tests, I'd suggest we create a Cypress command `cy.checkURL` that takes if the tests should be prefixed or not, then adapt.

**Comment by wesleybl** [comment]
@stevepiercy can you review it here please? What to do with the error:

https://github.com/plone/volto/actions/runs/5457862287/jobs/9932330404?pr=3719#step:7:156

?

The link in question is:

https://github.com/plone/volto/blob/e11931f4bda25ca92a35064e4546bb4940f18780/docs/source/theming/custom-styling.md?plain=1#L58

**Comment by stevepiercy** [comment]
For the error, someone should approve my open PR: https://github.com/plone/volto/pull/4941

File: docs/source/deploying/prefixed-root.md
Comment (stevepiercy) [inline docs/source/deploying/prefixed-root.md:3]:
```suggestion
  "description": "Prefixed (non-root) deployment for Volto"
```

File: docs/source/deploying/prefixed-root.md
Comment (stevepiercy) [inline docs/source/deploying/prefixed-root.md:4]:
```suggestion
  "property=og:description": "Prefixed (non-root) deployment for Volto"
```

File: docs/source/deploying/prefixed-root.md
Comment (stevepiercy) [inline docs/source/deploying/prefixed-root.md:2]:
```suggestion
myst:
  html_meta:
```

Also indent the subsequent lines 2 more spaces.

File: docs/source/deploying/prefixed-root.md
Comment (stevepiercy) [inline docs/source/deploying/prefixed-root.md:10]: In general, one sentence per line. Docs are not code. This makes review of a complete thought much easier. Please update.

File: docs/source/deploying/prefixed-root.md
Comment (stevepiercy) [inline docs/source/deploying/prefixed-root.md:13]:
```suggestion
If you're integrating a Volto website with another existing website, you may need to run Volto on a virtual path inside that website instead of the root path.
```

File: docs/source/deploying/prefixed-root.md
Comment (stevepiercy) [inline docs/source/deploying/prefixed-root.md:16]:
```suggestion
prefixed path of your Volto. For example, if I want Volto's root to be
```

File: docs/source/deploying/prefixed-root.md
Comment (stevepiercy) [inline docs/source/deploying/prefixed-root.md:41]: I think you added an extra path segment in the second `RewriteRule`. Please check.

```suggestion
RewriteRule ^/level-1(.*) http://volto:3000/level-1$$1 [P,L]
```

File: docs/source/deploying/prefixed-root.md
Comment (stevepiercy) [inline docs/source/deploying/prefixed-root.md:37]:
```suggestion
configure your rewrite rules to something like the following, in this case for Apache.
```

File: docs/source/deploying/prefixed-root.md
Comment (stevepiercy) [inline docs/source/deploying/prefixed-root.md:45]:
```suggestion
do the following.
Notice the multiple `_vh_` segments in the rewrite rule.
```

File: docs/source/deploying/prefixed-root.md
Comment (stevepiercy) [inline docs/source/deploying/prefixed-root.md:64]:
```suggestion
Finally, because `RAZZLE_PREFIX_PATH` is also used to configure the location of the static resources in webpack, when you build the static resources bundle, you must use a command such as the following.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Some English and MyST syntax and grammar fixes, and a style guide note about one line per sentence in docs. Thank you!

File: docs/source/deploying/prefixed-root.md
Comment (wesleybl) [reply docs/source/deploying/prefixed-root.md]: You are right. I made the change.

**Comment by wesleybl** [comment]
@stevepiercy I made the changes. Can you please have a look again?

**Review by stevepiercy — CHANGES_REQUESTED** [review]
A couple of corrections. It looks like a couple of bad copy-pastas created spaghetti docs. 🍝

**Comment by wesleybl** [comment]
@stevepiercy Done.

**Review by stevepiercy — APPROVED** [review]
Docs are good. Does this PR need code review by a maintainer?

**Comment by wesleybl** [comment]
> @wesleybl would you please merge master into this PR to get the docs to pass, now that the PR with a fix has been merged?

@stevepiercy done.

**Review by wesleybl — APPROVED** [review]
LGTM

**Comment by wesleybl** [comment]
@sneridagh @nileshgulia1 the tests are working now. Can we merge?

**Review by nileshgulia1 — APPROVED** [review]

**Comment by nileshgulia1** [comment]
Thanks @wesleybl for taking care!

**Comment by wesleybl** [comment]
This PR was merged into #3592

**Pull request merged**
